src/mame/drivers/1945kiii.cpp

```diff
@@ -365,16 +361,16 @@ static const gfx_layout k3_layout =
 	16,16,
 	RGN_FRAC(1,1),
 	8,
-	{ 0,1,2,3,4,5,6,7 },
-	{ 0*8,1*8,2*8,3*8,4*8,5*8,6*8,7*8,8*8,9*8,10*8,11*8,12*8,13*8,14*8,15*8 },
-	{ 0*128, 1*128, 2*128, 3*128, 4*128, 5*128, 6*128, 7*128, 8*128, 9*128, 10*128, 11*128, 12*128, 13*128, 14*128, 15*128 },
+	{ STEP8(0,1) },
+	{ STEP16(0,8) },
+	{ STEP16(0,8*16) },
 	16*128,
 };
 
 
 static GFXDECODE_START( gfx_1945kiii )
-	GFXDECODE_ENTRY( "gfx1", 0, k3_layout,   0x0, 2  ) /* bg tiles */
-	GFXDECODE_ENTRY( "gfx2", 0, k3_layout,   0x0, 2  ) /* bg tiles */
+	GFXDECODE_ENTRY( "gfx1", 0, k3_layout,   0x000, 2  ) /* sprites */
+	GFXDECODE_ENTRY( "gfx2", 0, k3_layout,   0x100, 1  ) /* bg tiles */
```

Comment:
This breaks bg GFX in a very evident way in every game of the driver. I'd really like to help merge your PRs, but I don't have time or interest to check every single game in the bigger drivers. So please, please test your changes locally before committing PRs.